processing/src/main/java/org/apache/druid/data/input/InputFormat.java

```diff
@@ -67,4 +69,22 @@ InputEntityReader createReader(
       InputEntity source,
       File temporaryDirectory
   );
+
+  /**
+   * Computes the weighted size of a given input object of the underyling input format type, weighted
+   * for its cost during ingestion. The weight calculated is dependent on the format type and compression type
+   * ({@link CompressionUtils.Format}) used if any. Uncompressed newline delimited json is used as baseline
+   * with scale factor 1. This means that when computing the byte weight that an uncompressed newline delimited
+   * json input object has towards ingestion, we take the file size as is, 1:1.
+   *
+   * @param compressionFormat The compression format of the input object, if any.
+   * @param size The size of the input object in bytes.
+   *
+   * @return The weighted size of the input object.
+   */
+  @JsonIgnore
```

Comment:
There should be no need for `@JsonIgnore` here. Typically our ObjectMapper is configured to ignore all methods that aren't explicitly annotated with `@JsonProperty`. Are you seeing something different?